packages/api-client/src/components/CommandPalette/TheCommandPalette.vue

```diff
@@ -52,30 +53,30 @@ import type { HotKeyEvent } from '@/libs'
 
 const modalState = useModal()
 const router = useRouter()
-const { activeWorkspace } = useActiveEntities()
-const { events } = useWorkspace()
+const { activeWorkspace, activeWorkspaceCollections } = useActiveEntities()
+const { events, requestMutators } = useWorkspace()
 
 /** Available Commands for the Command Palette */
 const availableCommands = [
   {
     label: '',
     commands: [
-      {
-        name: 'Create Request',
-        icon: 'ExternalLink',
-      },
       {
         name: 'Import from OpenAPI/Swagger/Postman',
         icon: 'Import',
       },
       {
-        name: 'Add Tag',
-        icon: 'Folder',
+        name: 'Create Draft Request',
```

Comment:
```suggestion
        name: 'Create Request',
```

I don’t think we have to explicitly mention that a new PR will land in the drafts collection